Gems/Atom/Feature/Common/Assets/Materials/Pipelines/Common/DepthPass.shader.template

```diff
@@ -1,5 +1,5 @@
 {
-    "Source" : "INSERT_AZSL_HERE",
+    "Source" : "INSERT_AZSL_PATH_HERE",
```

Comment:
This is a much better name. Thank you. I would still recommend having it use a default file path instead and removing the template extension. Besides allowing the AP to pick up and validate the hand edited file as is, material canvas allows you to open and edit shader files. 

Hopefully we can keep all of the new formats and changes to existing format simple enough that we can easily register material related data types as documents in material canvas, material editor, SMC, etc. 

I need to revisit some structures to make them a little more user friendly and add descriptions for all of the fields. 

We could also reregister the file type with the template extension. Please excuse the low res screenshots.

![image](https://user-images.githubusercontent.com/82461473/199353482-33099596-e753-4e7e-91c4-f607a638496c.png)
![image](https://user-images.githubusercontent.com/82461473/199353684-aa94aaea-200d-42f3-9a0c-8faf3f3069dc.png)

Reply:
It might work out the way you suggest at some point, I'll keep it in mind.